vic/drivers/cesm/src/display_current_settings.c

```diff
@@ -0,0 +1,454 @@
+/******************************************************************************
+ * @section DESCRIPTION
+ *
+ * This routine displays the current settings of options defined in the header
+ * files and the global parameter file.
+ *
+ * @section LICENSE
+ *
+ * The Variable Infiltration Capacity (VIC) macroscale hydrological model
+ * Copyright (C) 2014 The Land Surface Hydrology Group, Department of Civil
+ * and Environmental Engineering, University of Washington.
+ *
+ * The VIC model is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *****************************************************************************/
+
+#include <vic_driver_cesm.h>
+
+/******************************************************************************
+ * @brief    Display the current settings of options defined in the header
+ *           files and the global parameter file.
+ *****************************************************************************/
+void
+display_current_settings(int mode)
+{
```

Comment:
Again, some duplication of code here.  But I guess there's already an issue open about breaking up the code in this function into more atomic pieces and making most of them shared among drivers.